specification/storage/data-plane/Microsoft.BlobStorage/stable/2026-02-06/blob.json

```diff
@@ -12076,6 +12100,44 @@
       },
       "description": "Specify a SQL where clause on blob tags to operate only on blobs with a matching value."
     },
+    "IfBlobMatch": {
```

Comment:
Any reason you created new definitions for these instead of using the existing ones above? These ones don't include the `x-ms-parameter-grouping` which is important.